nvidia-gpu/nvidia-container-toolkit/nvidia-pkgs/pkg.yaml

```diff
@@ -32,16 +32,13 @@ steps:
         bash nvidia.run --extract-only
     install:
       - |
-        mkdir -p /rootfs/usr/local \
-          /rootfs/usr/local/lib/containers/nvidia-persistenced \
-          /rootfs/usr/local/etc/containers \
-          /rootfs/usr/etc/udev/rules.d
-
         cd NVIDIA-Linux-*
 
         ./nvidia-installer --silent \
           --opengl-prefix=/rootfs/usr/local \
           --utility-prefix=/rootfs/usr/local \
+          --utility-libdir=glibc/lib \
```

Comment:
this will only copy some libraries, not all hence the manual move below at https://github.com/siderolabs/extensions/pull/421/files#diff-d91dc320bde5625cb12535516b84cc3cd36e7ae24b320a40852b8f22a38299beR61

Reply:
for some reason `nvidia-container-cli` can't find libs stored under `/usr/local/lib` so we explicitly keep them under the custom location at `/usr/local/glibc/lib`